### ISSUE TITLE
can't see who received a distant message, if message has multiple recipients

### ISSUE THREAD

**Issue by electron128** [RetroShare/RetroShare#331]
Send a distant message with multiple receivers.
Outbox shows only the first recipient in the "to" column.
Send messages shows all recipients in the "to" column, but there is no way to see who has send an acknowledge.

Desired behavior:
- show the message in outbox until we have acknowledges from all receivers. Show the message only one time. Mark recipients which have send an acknowledge.
- show the message in sent folder (one time only) as soon as one ack was received. Mark recipients which did not send an acknowledge yet.

**Comment by csoler** [comment]
that's right! I noticed it as well. It's a bit problematic to show only one message: it needs to manage the list of destinations in outbox and in sentbox accordign to which message was ACKed or not. The minimum we could do to start with is to fix the list of destinations in the outbox.

**Comment by defnax** [comment]
@csoler ? close?

**Comment by csoler** [comment]
Distant messages UI has been re-implemented so that Outbox contains one copy for each recipient. So this should be fixed normally.
